src/main/java/dev/dfonline/codeclient/Utility.java

```diff
@@ -279,4 +279,19 @@ public static String textToString(Text content) {
         textToString(content, builder, GetActionDump.ColorMode.SECTION);
         return builder.toString();
     }
+
+    /**
+     * Generate a string of 32 random A-Z,a-z,0-9 characters that are used for authentication tokens in the API.
+     * @return A random authentication token.
+     */
+    public static String genAuthToken() {
+        String chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+        StringBuilder token = new StringBuilder();
+        Random random = new Random(System.currentTimeMillis());
```

Comment:
if I'm going to ask for changes I might as well:
maybe `SecureRandom`?
for that matter if you do use secure random you could condense this function down to
```java
SecureRandom rand = new SecureRandom();
byte[] randomBytes = new byte[16]; // or longer
random.nextBytes(randomBytes);
return HexFormat.of().formatHex(randomBytes); // or something else for a wider charset
```
or maybe I'm just nitpicking.